CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v1.1.0 - August 13, 2019
```

Comment:
One this that might be worth discussing is: Is a minor version correct for this? My thinking is that removing this rule from "recommended" won't break anyone's builds so I went with that.

Happy to go with something else if that's what we want.

Reply:
(marked as outdated because I updated the date to today)

Reply:
I think minor is fair. It won't break anything as lack of the `expect` rule won't cause new lint failures.

Also, I'm loathe to do v2 right after releasing v1

Reply:
@benkimpel  Thoughts?